README.md

```diff
@@ -91,8 +91,24 @@ Note: only supports Linux; no Windows or MacOS.
     ADD agent.py /agent.py
     ADD submission.sh /submission.sh
     ```
-    Build your docker container using: `docker build . --file Objectnav.Dockerfile  -t objectnav_submission`. (Note: you may need `sudo` priviliges to run this command.)
+    Build your docker container using: 
+    ```bash
+    docker build . --file Objectnav.Dockerfile  -t objectnav_submission
+    ```
+    
+    Note #1: you may need `sudo` priviliges to run this command.
+
+    Note #2: Please make sure that you keep your local version of `fairembodied/habitat-challenge:testing_2022_habitat_base_docker` image up to date with the image we have hosted on [dockerhub](https://hub.docker.com/r/fairembodied/habitat-challenge/tags). This can be done by pruning all cached images, using:
+    ```
+    docker system prune -a
+    ```
 
+1. Dataset: First, get access to the Habitat-Matterport3D Dataset scenes by visiting [this link](https://matterport.com/habitat-matterport-3d-research-dataset) and following the given instructions. After getting access to the dataset, carry out the following steps to download the dataset:
+    
+    a) First, you will need to generate a matterport API Token:
+
+<<<<<<< HEAD
```

Comment:
@dhruvbatra this should be removed.

Reply:
Yeah, apparently I can't git merge. Sigh. Fixed.